### PR TITLE
New version v1beta17

### PR DESCRIPTION
**Pull request by feloy**
Create a new version 
```
$ ./hack/new_version.sh
> v1beta17
```

to be able to change config in #3040

### CODE REVIEW

File: pkg/skaffold/schema/v1beta16/upgrade.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import (
+	next "github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util"
+	pkgutil "github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+)
+
+// Upgrade upgrades a configuration to the next version.
+// Config changes from v1beta16 to v1beta17
+// 1. Additions:
+//    buildArgs for Kustomize deployer
```
Comment (balopat) [inline pkg/skaffold/schema/v1beta16/upgrade.go]:
```suggestion
```

File: pkg/skaffold/deploy/kustomize.go [hunk 2]
```diff
@@ -108,6 +108,7 @@ func (k *KustomizeDeployer) Deploy(ctx context.Context, out io.Writer, builds []
 		event.DeployFailed(err)
 		return NewDeployErrorResult(errors.Wrap(err, "reading manifests"))
 	}
+	fmt.Printf("\n\n\n\n%v\n\n\n\n", manifests.String())
```
Comment (tejal29) [inline pkg/skaffold/deploy/kustomize.go]: remove this line

File: pkg/skaffold/deploy/kustomize.go [hunk 3]
```diff
@@ -139,7 +140,6 @@ func (k *KustomizeDeployer) Deploy(ctx context.Context, out io.Writer, builds []
 			return NewDeployErrorResult(errors.Wrap(err, "unable to transform manifests"))
 		}
 	}
-
```
Comment (tejal29) [inline pkg/skaffold/deploy/kustomize.go]: undo change

File: pkg/skaffold/deploy/kustomize.go [hunk 4]
```diff
@@ -190,7 +190,6 @@ func dependenciesForKustomization(dir string) ([]string, error) {
 	if err := yaml.Unmarshal(buf, &content); err != nil {
 		return nil, err
 	}
-
```
Comment (tejal29) [inline pkg/skaffold/deploy/kustomize.go]: undo change

**Comment by tejal29** [comment]
@feloy This looks good. Can you please revert some changes. Once its ready, i will review #3040

**Comment by tejal29** [comment]
@feloy looks like the linter failed. Can you please fix it.

**Comment by tejal29** [comment]
@feloy please address @balopat review.

**Pull request merged**